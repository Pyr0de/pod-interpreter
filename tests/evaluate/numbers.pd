10+10;
10-10;
100*12;
100/23;
0.1 + 0.2;
22/7;
100 > 100;
199 > 100;
100 >= 100;
100 == 100;
100 != 100;
100 < 99;
100 < 101;
100 <= 100;

10.2 > 10;
10.5 < 10;
10 <= 10.0;
10 < 1.3;

22.0/7.0;
12/5.2;
13 % 3;

13.0 % 3;

51.2/0.0;
13.0/0;

100 || 100;
100 && 100;
!100;
