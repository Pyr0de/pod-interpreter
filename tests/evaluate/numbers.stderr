ExitCode: 66
Cannot operator on "13.0 % 3"
[line 27] Error: Cannot divide by zero
[line 28] Error: Cannot divide by zero
Cannot operator on "100 || 100"
Cannot operator on "100 && 100"
Cannot operator on "! 100"
